sirepo/db_upgrade.py

```diff
@@ -85,6 +85,22 @@ def _20240507_cloudmc_to_openmc(qcall):
             _migrate_sim_type("cloudmc", "openmc", qcall, u)
 
 
+def _20240524_add_role_user(qcall):
+    if not qcall.auth_db.table_exists("user_role_invite_t"):
+        return
+    qcall.auth_db.execute_sql(
+        """
+INSERT INTO user_role_moderation_t (uid, role, status, moderator_uid, last_updated)
+SELECT uid, role, status, moderator_uid, last_updated
+FROM user_role_invite_t
+"""
+    )
+    qcall.auth_db.drop_table("user_role_invite_t")
+    for u in qcall.auth_db.all_uids():
```

Comment:
Since there's direct SQL above. I think it would better and simpler to write some SQL. There is an efficiency and ACID reason to do this. The shorter the transaction, the better.

```sql
INSERT INTO user_role_t (uid, role, expiration) 
VALUES ((select uid from user_registration_t), "user", NULL);
```

"user" could be an injected string, because it is a constant (no chance of sql injection attacks. This is an upgrade, and denormalizing the value makes it simpler to read and avoids the question of sql injection attack.

Reply:
[634efd8](https://github.com/radiasoft/sirepo/pull/7070/commits/634efd840a1338c81fced52736fe4cfde315bb63)